bom-weekly/pom.xml

```diff
@@ -251,6 +251,12 @@
         <artifactId>lockable-resources</artifactId>
         <version>1172.v4b_8fc8eed570</version>
       </dependency>
+      <dependency>
+        <groupId>org.jenkins-ci.main</groupId>
+        <artifactId>maven-plugin</artifactId>
+        <!-- TODO https://github.com/jenkinsci/maven-plugin/pull/313 -->
```

Comment:
Blocked on upstream release

Reply:
Maven plugin 3.23 has been released.  Tests pass with the 3.23 release.  I think this is ready to be taken out of draft once the `weekly` conditional is removed.